client/src/App.jsx

```diff
@@ -53,29 +51,23 @@ function App() {
       start_date: '2022-04-26',
       expiry_date: '',
       climbers: [],
-    }
+    },
   ];
 
-
   return (
-    <>
-   
+    <div className="App">
       <Routes>
-        <Route path="/" element={<Find bouldersList={bouldersList} />} />
+        <Route
+          path="/"
+          element={<Find bouldersList={bouldersList} title="Heelhook" />}
```

Comment:
I don't think that you need to define `title` here. You could handle this in your `Find` and `Add` component directly.